scrape/manager.go

```diff
@@ -130,6 +134,20 @@ func (m *Manager) reload() {
 	wg.Wait()
 }
 
+// setServerJitter calculates global jitter per server relying on extra label set.
+func (m *Manager) setServerJitter(labels model.LabelSet) error {
+	h := fnv.New64a()
+	hostname, err := os.Hostname()
```

Comment:
Could we use the fqdn here?

Reply:
what kind of problem we want to resolve with this? hostname is just an extra seed. I can add lookup, but it seems to be over-engineering in here.